lpa-submissions-e2e-tests/cypress/integration/appealsPlanningOfficersReport/appealsPlanningOfficersReportSteps.js

```diff
@@ -0,0 +1,125 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
+
+const pageTitle =
+  'Upload report to reach decision - Appeal Questionnaire - Appeal a householder planning decision - GOV.UK';
+const pageUrl = '/officers-report';
+const sectionName = 'Required documents';
+const validFileName = 'upload-file-valid.pdf';
+const completedStatusId = 'officers-report';
+const questionTitle = 'Planning Officer’s report';
+
+Given('LPA Planning Officer is within the LPA Questionnaire task list', () => {
+  cy.goToAppealsQuestionnaireTasklistPage();
+});
+
+Given('Upload the Planning Officers report is requested', () => {
+  cy.goToPage(pageUrl);
+  cy.verifyPage(pageUrl);
+  cy.verifyPageTitle(pageTitle);
+  cy.verifyQuestionTitle(questionTitle);
+  cy.verifySectionName(sectionName);
+});
+
+Given('a file has been uploaded', () => {
+  cy.goToPage(pageUrl);
+  cy.clickChooseFile(validFileName);
+  cy.clickUploadFile().click();
+  cy.validateFileUpload(validFileName);
+});
+
+Given('a file has been uploaded and confirmed', () => {
+  cy.goToUploadPlanningOfficersReportPage();
```

Comment:
This is an example of inconsistency that can be caused by deleting the initially written goTo methods for each page. You have deleted this but are still referring it here. Please add it as a separate refactor task and once prioritised, we can edit all the goTo methods at the same time.

Reply:
Edited in place here as refactors need to be on seperate stories.